presto-hive/pom.xml

```diff
@@ -516,6 +516,9 @@
                         <ignoredDependency>org.glassfish.jersey.core:jersey-common:jar</ignoredDependency>
                         <ignoredDependency>org.eclipse.jetty:jetty-server:jar</ignoredDependency>
                     </ignoredDependencies>
+                    <ignoredUnusedDeclaredDependencies>
+                        <ignoredUnusedDeclaredDependency>org.apache.lucene:lucene-core:jar:8.10.0</ignoredUnusedDeclaredDependency>
+                    </ignoredUnusedDeclaredDependencies>
```

Comment:
I also found this seems to be not needed? When I checked out the branch and built the entire repository with these lines removed, it worked fine.